The Fish game.md

```diff
@@ -0,0 +1,55 @@
+# Summary
+This task works to use collective sensing in groups of fish in order to better understand collective intelligence in human groups. To do this, individuals participated in a gradient-sensing task in which every location in this virtual world responded to a score value that changed over time.
+
+# References
+Main paper: https://people.csail.mit.edu/pkrafft/papers/krafft-et-al-2015-emergent.pdf
+
+Supporting paper: https://www-jstor-org.proxy.library.upenn.edu/stable/pdf/23343730.pdf?refreqid=excelsior%3A497fb73bd3cb7e6d7fbdb013b9e0a0f4
+
+# Stimuli
+## The visual components
+![Fish1](images/Fish1.png)
+![Fish2](images/Fish2.png)
+
```

Comment:
Should the scorefields be included: 
 These score fields consisted of 480 × 285 arrays of score values for each 125ms time interval in our game.  First, a “spotlight” of high value was created that moved in straight paths between uniformly randomly chosen locations. This spotlight was then combined with a field of spatially correlated noise. This procedure yields a complex landscape with many transient maxima and a single persistent time-varying global maximum.